SimpleWAWS/Controllers/ResourceController.cs

```diff
@@ -200,19 +209,18 @@ public Task<HttpResponseMessage> All()
                         resourceGroup = await resourceManager.ActivateLogicApp(template as LogicTemplate, identity, anonymousUserName);
                         break;
                     case AppService.Function:
-                        if (identity.Issuer == "OrgId")
-                        {
-                            return Request.CreateErrorResponse(HttpStatusCode.BadRequest, Resources.Server.Error_OrgIdNotSupported);
-                        }
-                        else if (identity.Issuer != "MSA")
-                        {
-                            return SecurityManager.RedirectToAAD(template.CreateQueryString());
-                        }
+                        //if (identity.Issuer == "OrgId")
+                        //{
+                        //    return Request.CreateErrorResponse(HttpStatusCode.BadRequest, Resources.Server.Error_OrgIdNotSupported);
+                        //}
+                        //else if (identity.Issuer != "MSA")
+                        //{
+                        //    return SecurityManager.RedirectToAAD(template.CreateQueryString());
+                        //}
```

Comment:
I'll remove the commented code out.
I dont know why the code is needed API and LogicApps ?